CMakeLists.txt

```diff
@@ -209,7 +209,7 @@ endif(CMAKE_BUILD_TYPE MATCHES FullDebug)
 
 ##*****************************
 #find and include OpenMP if possible
-message(KRATOS_EXCLUDE_OPENMP ${KRATOS_EXCLUDE_OPENMP})
+#message(STATUS KRATOS_EXCLUDE_OPENMP : ${KRATOS_EXCLUDE_OPENMP})
```

Comment:
please remove commented code
this confuses if one later reads the code

+1 for the removal of this line

Reply:
No problem, I think this file must be reviewed deeply later on. Usually this variable ${KRATOS_EXCLUDE_OPENMP} is empty and prints nothing. I will delete this line.